backend-rust/README.md

```diff
@@ -134,6 +156,46 @@ Variables:
 
 ![ExampleQuery](./ExampleQuery.png)
 
+### Small services included in the GraphQL API Service:
+
+#### Monitoring:
+
+When the GraphQL API service is run, it exposes a metric endpoint [metric endpoint](https://github.com/Concordium/concordium-scan/blob/main/backend-rust/src/router.rs):
+
+```
+http://localhost:8003/metrics
+```
+
+This endpoint is scraped by Prometheus, and the collected metrics are visualized in Grafana dashboards for internal monitoring.
+
+#### Rest API server:
+
+When the GraphQL API service is run, it exposes rest [API (legacy) endpoints](https://github.com/Concordium/concordium-scan/blob/main/backend-rust/src/rest_api.rs):
+
+These legacy api endpoints need to be maintained for backwards compatibility since several external partners (e.g. `coingecko`/`coinmarketcap`) scrap the total CCD supply in circulation from these endpoints.
```

Comment:
I would prefer not naming any external service here and just vaguely mention external partners.